Client/game_sa/CModelInfoSA.cpp

```diff
@@ -1493,7 +1498,7 @@ void CModelInfoSA::MakeVehicleAutomobile(ushort usBaseID)
     m_pInterface->usNumberOfRefs = 0;
     m_pInterface->pRwObject = nullptr;
     m_pInterface->pVisualInfo = nullptr;
-    m_pInterface->usUnknown = 65535;
+    m_pInterface->uObjectInfoIndex = 65535;
```

Comment:
As above

Reply:
correcting flag name

Reply:
> As above

The whole flags order of CBaseModelInfoSAInterface is at a mess 😂

Reply:
but is this necessary to fix the collision glitch? if not, it needs to be in a separate PR